core/client/views/settings/navigation.js

```diff
@@ -1,5 +1,12 @@
 import BaseView from 'ghost/views/settings/content-base';
 
-var SettingsNavigationView = BaseView.extend();
+var SettingsNavigationView = BaseView.extend({
+    keyDown: function (event) {
+        if (event.keyCode === 187) {
```

Comment:
Oh, just noticed the equal sign part.  That's an issue.  Needs to only fire on `+`.